### ISSUE TITLE
Add element ID to log messages

### ISSUE THREAD

**Issue by carlcs** [verbb/feed-me#406]
It would be really helpful if the created element’s ID would be noted in the log messages. Something like “Entry 11432 added (element ID 72636).”

**Comment by engram-design** [comment]
Not sure If I'm missing something here - but aren't they the same thing? An Entry's ID is the Element ID? Right now, logs show the added/updated Element ID.

**Comment by carlcs** [comment]
You are right, it’s the element ID that is logged. For some reason I though it is the feed’s item number. The intent of this feature request was to make it possible to see both in the logs, to get an idea of the mapping, something like “My Feed Name (#11432) – Entry 72636 added.“

**Comment by engram-design** [comment]
Gotcha. Only the item is shown on error, so I'll add that to everything.
